showcase/src/assets/translations/nl.json

```diff
@@ -43,9 +43,11 @@
       },
       "DATE_PICKER": {
         "CUSTOM_FILTER": "Date Picker - Aangepaste filter",
+        "NG_MODEL": "Date Picker - NgModel gebruiken",
```

Comment:
A literal translation would be "gebruik makend van NgModel" but  I think "Met NgModel" fits better here